include/NifFile.hpp

```diff
@@ -153,7 +153,7 @@ class NifFile {
 
 	std::vector<std::string> GetShapeNames() const;
 	std::vector<NiShape*> GetShapes() const;
-	bool RenameShape(NiShape* shape, const std::string& newName);
+	static bool RenameShape(NiShape* shape, const std::string& newName);
```

Comment:
A lot of the functions you turned static should or can be located elsewhere instead (for example in the block classes, after changing some more `std::string shapeName` parameters to `NiShape*`).

Would you mind changing the static back to const for the time being, so that I can do a refactor on the functions that would be considered static at some point? Right now it just makes it inconsistent.

Reply:
I understand, but I think it is better to leave them static. That way, it will be easy to locate them once it is time to move them